### PR TITLE
added pod disruption budget

### PR DESCRIPTION
**Pull request by yurymuski**
PDB is critical for production setups where where replicaCount > 1 or autoscaling enabled

### CODE REVIEW

**Comment by yurymuski** [comment]
Hi @FZambia please review :)

File: charts/centrifugo/templates/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{{- if or (gt (.Values.replicaCount | int) 1) (gt (.Values.autoscaling.minReplicas | int) 1) }}
+apiVersion: {{ ternary "policy/v1" "policy/v1beta1" (semverCompare ">=1.21.0-0" .Capabilities.KubeVersion.Version) }}
```
Comment (FZambia) [inline charts/centrifugo/templates/poddisruptionbudget.yaml:2]: Do you personally need `policy/v1beta1`? If not I think we can go with `policy/v1` only

File: charts/centrifugo/values.yaml [hunk 2]
```diff
@@ -161,6 +161,9 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
+pdb:
```
Comment (FZambia) [inline charts/centrifugo/values.yaml:164]: Let's call it podDisruptionBudget

**Comment by yurymuski** [comment]
resolved

File: charts/centrifugo/values.yaml [hunk 3]
```diff
@@ -161,6 +161,10 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
+poddisruptionbudget:
```
Comment (FZambia) [inline charts/centrifugo/values.yaml:164]: `podDisruptionBudget`

Comment (yurymuski) [reply charts/centrifugo/values.yaml:164]: :D pdb was so handy)) will push tomorrow

Comment (FZambia) [reply charts/centrifugo/values.yaml:164]: sorry :) [Python Debugger](https://docs.python.org/3/library/pdb.html) for me :)

**Comment by yurymuski** [comment]
@FZambia  resolved :)

**Comment by FZambia** [comment]
Thanks Yury!

**Pull request merged**